src/shared/ui/survey/Slider/SurveySlider.tsx

```diff
@@ -37,7 +37,7 @@ const SurveySlider: FC<SliderProps> = ({ config, ...props }) => {
   return (
     <YStack>
       <Slider
-        defaultValue={[initialValue]}
+        value={initialValue != null ? [initialValue] : undefined}
```

Comment:
Can we avoid "undefined" for value's value? Use null if value not set? Code would be more robust if we do not mess undefined with null. If not possible then ignore

Reply:
I would love to do that but `undefined` is expected as a "no value" by the third-party library in that case.